modules/pipeline/providers/edgepipeline_register/provider.go

```diff
@@ -36,7 +38,7 @@ type Config struct {
 	ClusterDialEndpoint        string        `file:"cluster_dialer_endpoint" desc:"cluster dialer endpoint"`
 	ClusterAccessKey           string        `file:"cluster_access_key" desc:"cluster access key, if specified will doesn't start watcher"`
 	RetryConnectDialerInterval time.Duration `file:"retry_cluster_hook_interval" default:"1s"`
-	AccessToken                string        `file:"token" env:"EDGE_PIPELINE_ACCESS_TOKEN"`
+	AccessClusterEtcdKeyPrefix string        `file:"edge_pipeline_access_cluster_etcd_key_prefix" env:"EDGE_PIPELINE_ACCESS_CLUSTER_ETCD_KEY_PREFIX" default:"/devops/pipeline/v2/edge/pipeline/access-key"`
```

Comment:
no need default value here.
only define once in config.yaml or provider.config.